### DISCUSSION TITLE
Whlie Flash Wifi Example it stops at 99%

### DISCUSSION THREAD

**Discussion by AbirahamLincoln** [bitcraze/discussions#159] [Q&A]
cfloader flash aideck_gap8_wifi_img_streamer_with_ap.bin deck-bcAI:gap8-fw -w radio://0/80/2M/E7E7E7E7E7
Reset to bootloader mode ...
Could not save cache, no writable directory
Could not save cache, no writable directory
Skipping bcAI:esp, not in the target list
Deck bcAI:gap8, reset to bootloader
| 0% Writing to bcAI:gap8 deck memory
/ 1% Writing to bcAI:gap8 deck memory
- 2% Writing to bcAI:gap8 deck memory
\ 3% Writing to bcAI:gap8 deck memory
| 4% Writing to bcAI:gap8 deck memory
/ 5% Writing to bcAI:gap8 deck memory
- 6% Writing to bcAI:gap8 deck memory
\ 7% Writing to bcAI:gap8 deck memory
| 8% Writing to bcAI:gap8 deck memory
/ 9% Writing to bcAI:gap8 deck memory
- 10% Writing to bcAI:gap8 deck memory
\ 11% Writing to bcAI:gap8 deck memory
| 12% Writing to bcAI:gap8 deck memory
/ 13% Writing to bcAI:gap8 deck memory
- 14% Writing to bcAI:gap8 deck memory
\ 15% Writing to bcAI:gap8 deck memory
| 16% Writing to bcAI:gap8 deck memory
/ 17% Writing to bcAI:gap8 deck memory
- 18% Writing to bcAI:gap8 deck memory
\ 19% Writing to bcAI:gap8 deck memory
| 20% Writing to bcAI:gap8 deck memory
/ 21% Writing to bcAI:gap8 deck memory
- 22% Writing to bcAI:gap8 deck memory
\ 23% Writing to bcAI:gap8 deck memory
| 24% Writing to bcAI:gap8 deck memory
/ 25% Writing to bcAI:gap8 deck memory
- 26% Writing to bcAI:gap8 deck memory
\ 27% Writing to bcAI:gap8 deck memory
| 28% Writing to bcAI:gap8 deck memory
/ 29% Writing to bcAI:gap8 deck memory
- 30% Writing to bcAI:gap8 deck memory
\ 31% Writing to bcAI:gap8 deck memory
| 32% Writing to bcAI:gap8 deck memory
/ 33% Writing to bcAI:gap8 deck memory
- 34% Writing to bcAI:gap8 deck memory
\ 35% Writing to bcAI:gap8 deck memory
| 36% Writing to bcAI:gap8 deck memory
/ 37% Writing to bcAI:gap8 deck memory
- 38% Writing to bcAI:gap8 deck memory
\ 39% Writing to bcAI:gap8 deck memory
| 40% Writing to bcAI:gap8 deck memory
/ 41% Writing to bcAI:gap8 deck memory
- 42% Writing to bcAI:gap8 deck memory
\ 43% Writing to bcAI:gap8 deck memory
| 44% Writing to bcAI:gap8 deck memory
/ 45% Writing to bcAI:gap8 deck memory
- 46% Writing to bcAI:gap8 deck memory
\ 47% Writing to bcAI:gap8 deck memory
| 48% Writing to bcAI:gap8 deck memory
/ 49% Writing to bcAI:gap8 deck memory
- 50% Writing to bcAI:gap8 deck memory
\ 51% Writing to bcAI:gap8 deck memory
| 52% Writing to bcAI:gap8 deck memory
/ 53% Writing to bcAI:gap8 deck memory
- 54% Writing to bcAI:gap8 deck memory
\ 55% Writing to bcAI:gap8 deck memory
| 56% Writing to bcAI:gap8 deck memory
/ 57% Writing to bcAI:gap8 deck memory
- 58% Writing to bcAI:gap8 deck memory
\ 59% Writing to bcAI:gap8 deck memory
| 60% Writing to bcAI:gap8 deck memory
/ 61% Writing to bcAI:gap8 deck memory
- 62% Writing to bcAI:gap8 deck memory
\ 63% Writing to bcAI:gap8 deck memory
| 64% Writing to bcAI:gap8 deck memory
/ 65% Writing to bcAI:gap8 deck memory
- 66% Writing to bcAI:gap8 deck memory
\ 67% Writing to bcAI:gap8 deck memory
| 68% Writing to bcAI:gap8 deck memory
/ 69% Writing to bcAI:gap8 deck memory
- 70% Writing to bcAI:gap8 deck memory
\ 71% Writing to bcAI:gap8 deck memory
| 72% Writing to bcAI:gap8 deck memory
/ 73% Writing to bcAI:gap8 deck memory
- 74% Writing to bcAI:gap8 deck memory
\ 75% Writing to bcAI:gap8 deck memory
| 76% Writing to bcAI:gap8 deck memory
/ 77% Writing to bcAI:gap8 deck memory
- 78% Writing to bcAI:gap8 deck memory
\ 79% Writing to bcAI:gap8 deck memory
| 80% Writing to bcAI:gap8 deck memory
/ 81% Writing to bcAI:gap8 deck memory
- 82% Writing to bcAI:gap8 deck memory
\ 83% Writing to bcAI:gap8 deck memory
| 84% Writing to bcAI:gap8 deck memory
/ 85% Writing to bcAI:gap8 deck memory
- 86% Writing to bcAI:gap8 deck memory
\ 87% Writing to bcAI:gap8 deck memory
| 88% Writing to bcAI:gap8 deck memory
/ 89% Writing to bcAI:gap8 deck memory
- 90% Writing to bcAI:gap8 deck memory
\ 91% Writing to bcAI:gap8 deck memory
| 92% Writing to bcAI:gap8 deck memory
/ 93% Writing to bcAI:gap8 deck memory
- 94% Writing to bcAI:gap8 deck memory
\ 95% Writing to bcAI:gap8 deck memory
| 96% Writing to bcAI:gap8 deck memory
/ 97% Writing to bcAI:gap8 deck memory
- 98% Writing to bcAI:gap8 deck memory
\ 99% Writing to bcAI:gap8 deck memory

It stops at 99% ...

**Comment by knmcguire** [comment]
Hi!

I assume you went through [this tutorial](https://www.bitcraze.io/documentation/tutorials/getting-started-with-aideck/) right?

Which version of the AIdeck 1.1 do you have? You can find the revision letter on the bottom of the deck.

Here are some things you can try:

- You can try out the solution presented in this discussion before: https://github.com/orgs/bitcraze/discussions/57
- Also reflashing the ESP [via these instructions sometimes](https://www.bitcraze.io/documentation/tutorials/getting-started-with-aideck/#update-crazyflie-and-aideck-firmware) helps as well.

**Comment by AbirahamLincoln** [comment]
I have tried the Getting started many times.But the ai deck bootloader
stops at 99% only .

Please help with this.

**Reply by knmcguire** [reply]
Could you please answer the rest of the question I had above?

1. Which revision of the AIdeck 1.1 do you have
2. Have you tried the solution of the discussion here: https://github.com/orgs/bitcraze/discussions/57
3. Let me reframe my earlier question about the ESP reflashing: Did at any point in the [tutorial](https://www.bitcraze.io/documentation/tutorials/getting-started-with-aideck/), something did not go as expected? 
  a. Example: Maybe didn't see `ESP32: I (910) SYS: Initialized` in the console tab in the cfclient [after this step](https://www.bitcraze.io/documentation/tutorials/getting-started-with-aideck/#update-crazyflie-and-aideck-firmware)

**Reply by AbirahamLincoln** [reply]
I see ESP32: I (910) SYS: Initialized in the console tab

**Reply by AbirahamLincoln** [reply]
should i do this step?
['Gap8 bootloader (for older AIdecks revisions A-C) ](https://www.bitcraze.io/documentation/tutorials/getting-started-with-aideck/#:~:text=the%20steps%20again.-,Gap8%20bootloader%20(for%20older%20AIdecks%20revisions%20A%2DC),-If%20you%20do)

**Reply by AbirahamLincoln** [reply]
And also i can't find aideck examble wifi in my my system.

**Reply by AbirahamLincoln** [reply]
I have AI deck 1.1

**Comment by AbirahamLincoln** [comment]
Yes.

1. I have AIdeck 1.1
2. I tried the solution of the discussion .
https://github.com/orgs/bitcraze/discussions/57
3.I saw ESP32: I (910) SYS: Initialized in the console.

But the flashing stops at 99%.

**Reply by knmcguire** [reply]
![image](https://user-images.githubusercontent.com/11061260/181446016-3b17dc6b-a599-4038-9ea3-115dea098381.png)
Here is where you can find the revision of the AIdeck 1.1

**Comment by AbirahamLincoln** [comment]
Yes I have Rev C1

**Comment by knmcguire** [comment]
Since you have an older revision of the aideck, which is anything lower than rev. D, then you will need to flash the gap8 bootloader in order to get the over air flashing to work.  You'll need an jtag cable programmer for that work but the tutorial will point you to the right page. A jlink programmer has been reported to work as well.

['Gap8 bootloader (for older AIdecks revisions A-C)](https://www.bitcraze.io/documentation/tutorials/getting-started-with-aideck/#:~:text=the%20steps%20again.-,Gap8%20bootloader%20(for%20older%20AIdecks%20revisions%20A%2DC),-If%20you%20do)

**Comment by AbirahamLincoln** [comment]
Respected sir/madam,

Thank you so much for the information we will sure check it out and get
back to you as soon as possible with a updation.

Thank you
